### ISSUE TITLE
Enable twisted.conch.test.test_helper on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8851]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8851|
|Type|enhancement|
|Created|2016-10-04 20:13:44Z|
|Branch|https://github.com/twisted/twisted/tree/8851-rodrigc-conch|

https://github.com/twisted/twisted/pull/554

<details><summary>Searchable metadata</summary>

```
trac-id__8851 8851
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8851_rodrigc_conch 8851-rodrigc-conch
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1475612024850905 1475612024850905
changetime__1475837360252857 1475837360252857
version__None None
owner__Craig_Rodrigues__rodrigc_____ Craig Rodrigues <rodrigc@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_3"></a>|@acabhishek942 commented|
|-|-|

looks good to me. Please merge

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc set owner to @rodrigc<br>@rodrigc set status to `closed`|
|-|-|

In changeset 93c23565953b6938a90bc34cc5a5b646fd8acbbf
```
#!CommitTicketReference repository="" revision="93c23565953b6938a90bc34cc5a5b646fd8acbbf"
Merge pull request #554 from twisted/8851-rodrigc-conch

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8851

Enable twisted.conch.test.test_helper on Python 3
```
